generic/pom.xml

```diff
@@ -219,7 +219,7 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.5</version>
+            <version>[4.5.13,)</version>
```

Comment:
Thoughts for learning purposes, this means "versions after 4.5.13 (4.5.13 included)"? The soft bracket however indicates "exclusive"? So will it include or exclude the later version in that case? I.e. what happens if you write `[4.5.13,]`?

Reply:
here's a good reference https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN402